### PR TITLE
support python sdk for ormb

### LINKED ISSUE #34: [feature] Provide Python SDK
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind feature

**What happened**:

We need to provide Python SDK like https://docker-py.readthedocs.io/en/stable/ to support CURD in Jupyter Notebook or Python code directly.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

### PR DESCRIPTION
**Pull request by ZhuYuJin**
**What this PR does / why we need it**:

[feature] Provide Python SDK

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #34

Reference to #34

**Special notes for your reviewer**:

/cc @gaocegege 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: extern-sdk/python/ormb/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from ormb.api import *
+
+
+__version__ = '0.0.1'
```
Comment (gaocegege) [inline extern-sdk/python/ormb/__init__.py:4]: Can we add pyc into gitignore?

File: extern-sdk/python/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+requests
```
Comment (gaocegege) [inline extern-sdk/python/requirements.txt:1]: Can we add version for the requirement?

File: extern-sdk/python/ormb/__init__.py (hunk 1 above)
Comment (ZhuYuJin) [reply extern-sdk/python/ormb/__init__.py:4]: sure

File: extern-sdk/python/requirements.txt (hunk 2 above)
Comment (ZhuYuJin) [reply extern-sdk/python/requirements.txt:1]: sure

**Pull request merged**